### ISSUE TITLE
"Confirmed working" updates

### ISSUE THREAD

**Issue by teambvd** [bryanvaz/vfnet#7]
Didn't want to presume the addition of a pull request from myself, so just thought to update here instead -

In short, this should work with *all* boards utilizing the x500 and x700 chipsets - I've used them on all of mine, inc. X11SPM (X722 chipset), X10SRM (X550), and several others.

Something that'd be worth adding is that, if someone's already created persistent rules, depending on how those were generated, they may override those created by vfnet. Udev rules specifically, but possibly others as well. They'd need to be cleared manually if in use in order to use vfnet properly.

Great tool, and thanks for the efforts here!

**Comment by bryanvaz** [comment]
> In short, this should work with _all_ boards utilizing the x500 and x700 chipsets - I've used them on all of mine, inc. X11SPM (X722 chipset), X10SRM (X550), and several others.

So you tested it with your x500 and x700 boards and it provisions and deprovisions the Virtual Functions without any issues? Which NICs/Controllers were you using?

> Something that'd be worth adding is that, if someone's already created persistent rules, depending on how those were generated, they may override those created by vfnet. Udev rules specifically, but possibly others as well. They'd need to be cleared manually if in use in order to use vfnet properly.

Oh yea that's totally a thing, but only if you use the persistence feature.

> Great tool, and thanks for the efforts here!

Have you noticed any issues with it? Any ideas for improvement? A month ago it was misbehaving and not creating the VFs on reboot, but I never found the source and it just started working again.

**Comment by teambvd** [comment]
> So you tested it with your x500 and x700 boards and it provisions and deprovisions the Virtual Functions without any issues? Which NICs/Controllers were you using?

Yup! The motherboards mentioned have the aforementioned chipsets onboard (X11SPM motherboard uses X722 chipset, X10SRM has an onboard X550).

> Have you noticed any issues with it? Any ideas for improvement? A month ago it was misbehaving and not creating the VFs on reboot, but I never found the source and it just started working again.

No issues here, been rock solid across five boards. To really make this tool shine, you could implement functions for the igbe and mlnx drivers I suppose? igbe is easy enough, just needs to call `sriov_maxvfs` instead of `sriov_numvfs`. Mellanox is a bit more, calling `num_vfs`, but across different drivers, and there's some firmware wonkiness with some of them as well which could lead to support nightmares with users unaware of such, so not sure there.
